README.md

```diff
@@ -50,6 +50,28 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described [here](https://sanctuary.js.org/#types). In summary:
+
+1. New types can be created via type constructors. Type constructors take one or more type arguments. For example, the type "array of strings" is written as `Array String`, and the type constructor `Array` can be defined as `Array :: a -> a`.
```

Comment:
Thanks David. I'm still grokking this - I think I need to read up a bit more on type constructors (looking at Sanctuary).

I do get this though:

> Array :: a -> a would be an oddly named identity function

so `Array :: a -> a` is wrong - but I'm still looking into why `Array :: Type -> Type` is right.